packages/lib/slots.ts

```diff
@@ -169,7 +169,7 @@ function buildSlotsWithDateRanges({
       ? range.start
       : startTimeWithMinNotice;
 
-    let interval = 15;
+    let interval = Number(process.env.NEXT_PUBLIC_AVAILABILITY_SCHEDULE_INTERVAL) || 15;
```

Comment:
I assumed this 15 minute interval likely relates to the 15 minutes on availability.

Reply:
I'm not sure this interval does anything tbh - just a good default; @CarinaWolli could you have a look at this line?

Reply:
The interval setting ensures that available slots start only at specific times: on the hour, half-hour, or at 20-minute, 10-minute, or the default 15-minute intervals, depending on the event length.

Example: 
- NEXT_PUBLIC_AVAILABILITY_SCHEDULE_INTERVAL = 1
- Event length: 8 mins
- Availability starts at 9:37 => Yes, we only allowed 15-minute intervals before, but the incoming date range could still start at a random time because of a calendar busy time or the current day's availability is also usually random.

Before: 
First slot: 9:45 => starting by default at the next 15 minute slot

With these changes: 
First slot: 9:37

Reply:
Ok, @CarinaWolli   if our desired config is to use NEXT_PUBLIC_AVAILABILITY_SCHEDULE_INTERVAL = 10. 

And our desired first availability slot is to start at 9.10am.   How would this behave?  
Or would not changing this line at all still result in a 9.10 first booking?

Reply:
or are you saying that leaving this at 15 would have no impact?      ie-  event length of 10mins, and a users availability start time of 9.10 would show up correctly with 9.10 as the first available booking time?

Reply:
@CarinaWolli  or are you saying that leaving this at 15 would have no impact? ie- event length of 10mins, and a users availability allows 10 minute intervals and starts their day at 9.10am. it would show up correctly with 9.10am as the first available booking time?

Reply:
> Ok, @CarinaWolli if our desired config is to use NEXT_PUBLIC_AVAILABILITY_SCHEDULE_INTERVAL = 10.
> And our desired first availability slot is to start at 9.10am. How would this behave?
> Or would not changing this line at all still result in a 9.10 first booking?

In this case, we could leave it as 15. Because 10 is part of the `intervalsWithDefinedStartTimes` array. But if your NEXT_PUBLIC_AVAILABILITY_SCHEDULE_INTERVAL=1 and you have an availability starting at 8:13, then the first slot would not be 8:13 but 8:15.

I think we should keep your changes like that, it makes sense to me